src/modify_resources.py

```diff
@@ -82,232 +86,239 @@ def remove_all_network_manipulation(ssh_client, remove_all_machines=False):
-	                for container in container_ids:
-		                cmd = "sudo tc qdisc del dev {} root".format(container)
-		                ssh_exec(temp_ssh_client, cmd)
+                    temp_ssh_client = quilt_ssh(machine)
+                    container_ids = get_container_id(temp_ssh_client, full_id=False, append_c=False)
+                    for container in container_ids:
+                        cmd = "sudo tc qdisc del dev {} root".format(container)
+                        ssh_exec(temp_ssh_client, cmd)
 
 '''Stressing the CPU'''
 #Indicates the number of 10%-stress that is introduced into the system
 def update_cpu_through_stress(ssh_client, num_stress=0):
-        """CPU Limit describes how much CPU will be used by the stresser"""
-	if num_stress < 0:
-		invalid_resource_parameter("CPU Allocation", limit)
-		return
+    """CPU Limit describes how much CPU will be used by the stresser"""
+    if num_stress < 0:
+        invalid_resource_parameter("CPU Allocation", limit)
+        return
         
         #Separate CPU limit into several smaller increments of 10 instead of one larger one
-        for x in range(num_stress):
-	        cmd = "stress -c 1 &> /dev/null & cpulimit -p $( pidof -o $! stress ) -l {} &> /dev/null &".format(10)
-	        ssh_exec(ssh_client, cmd)
+    for x in range(num_stress):
+        cmd = "stress -c 1 &> /dev/null & cpulimit -p $( pidof -o $! stress ) -l {} &> /dev/null &".format(10)
+        ssh_exec(ssh_client, cmd)
 
 #Allows you to set the CPU periods in a container, intended for a container that is already running
 #as opposed to update_cpu which actually just stresses the CPU by a predetermined amount
 #Assumes that that CPU_period and CPU_quota were not set beforehand
 #CPU period is assumed to be 1 second no matter what
 def set_cpu_shares(ssh_client, cpu_quota):
-        container_names = get_container_names(ssh_client)
-        cpu_quota = int(cpu_quota * get_num_cores(ssh_client) * 1000000)
-        cpu_period = 1000000 #1 second in microseconds
-
-        #CPU Quota must be less than 1 second
-        while cpu_quota > 1000000:
-                cpu_quota = cpu_quota/2
-                cpu_period = cpu_period/2
-
-        print 'Adjusted CPU period: {}'.format(cpu_period)
-        print 'CPU Quota: {}'.format(cpu_quota)
-
-        throttled_containers = []
-        
-        for container_name in container_names:
-                if container_name not in container_blacklist:
-                        update_command = 'docker update --cpu-period {} --cpu-quota {} {}'.format(cpu_period, cpu_quota, container_name)
-                        print update_command
-                        ssh_client.exec_command(update_command)
-                        throttled_containers.append(container_name)
+    container_names = get_container_names(ssh_client)
+    # cpu_quota = int(cpu_quota * get_num_cores(ssh_client) * 1000000)
+    cpu_quota = int((float(cpu_quota) / 10) * 1000000)
```

Comment:
Isn't there some assumption about the CPU quota being some multiple of the number of cores in the machine?